security/pkg/server/ca/server_test.go

```diff
@@ -142,7 +142,85 @@ func TestExtractRootCertExpiryTimestamp(t *testing.T) {
 		},
 	}
 	for _, tc := range testCases {
-		sec := extractRootCertExpiryTimestamp(ca) - float64(time.Now().Unix())
+		sec := extractRootCertExpiryTimestamp(ca.GetCAKeyCertBundle()) - float64(time.Now().Unix())
+		if sec < tc.ttlRange[0] || sec > tc.ttlRange[1] {
+			t.Errorf("[%v] Failed, expect within range [%v, %v], got %v", tc.name, tc.ttlRange[0], tc.ttlRange[1], sec)
+		}
+		if tc.sleep != 0 {
```

Comment:
shouldn't we sleep before calling extractRootCertExpiryTimestamp()?

Reply:
to be honest, I haven't changed this test, I've just changed the params that we are passing to this function.